### ISSUE TITLE
Include template name in project details panel

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1200]
I've been asked how people can tell (without previewing or opening the editor) which template they have used for each project listed in their workspace. At the moment, the only way you can tell is by looking at the icon next to the project name in the workspace tree. This isn't accessible. 

It would be good to add 'Template: Xerte Online Toolkits/Bootstrap' to the project details panel

**Comment by torinfo** [comment]
@FayCross I'll add that (as I am messing with the Project details anyway)
And good idea, I think it would be good to add the name of the subtemplate (if used). (There is no other way to see wich sub template was used). Agree?

**Comment by FayCross** [comment]
@torinfo thanks, including subtemplate makes sense too.

**Comment by ronm123** [comment]
@FayCross @torinfo agree its a good idea to show the template name and sub template name but I don't think it needs to be prefixed by Xerte Online Toolkits. I think that's a bit redundant and as we know most of the time people refer to just Xerte  not Xerte Online Toolkits. Also in my experience sub-template names can be quite long in some cases so I think it would be neater and not necessary to include that prefix. Or am I missing something?

**Comment by ronm123** [comment]
@FayCross @torinfo just to add to this I would suggest the following format:
Template: Xerte Online Toolkit
Template: Bootstrap
Template: Decision Tree
Template: Sub Template Name

But is there a question/issue about where this is picked from? e.g. isn't there variation where the names have been customised via management like Bootstrap/Site Template rather than just Bootstrap.
Also does the sub template name need to be Template: Xerte Online Toolkit/Sub Template Name e.g. to indicate the parent.
And what happens when a sub template is removed - what happens to the name then?

**Comment by torinfo** [comment]
@ronm123 Good questions:
About where to take the name from. Probably the customised name? 
About sub templates, I think it would be good to name parent as well.

**Comment by FayCross** [comment]
@ronm123 I don't think it should be prefixed by Xerte Online Toolkits. I meant it should say Template: Xerte Online Toolkit or Template: Bootstrap (I accidentally added an s to the end of the XOT template name!) If we could start over it would be a lot less confusing if both XOT & Bootstrap templates were named differently!

We don't use sub-templates so can't comment on best way to show these.

**Comment by ronm123** [comment]
@FayCross yeah sorry I guessed after my first reply that you probably meant that. Also yes agree if we could start over it would be simple and I think one of the most confusing bits of that is the Xerte Online Toolkits Xerte Online Toolkit template! I might be remembering wrongly but I also seem to recall that the name of the Bootstrap template was named that way rather than simply e.g. Website template due to political issues at Nottingham and the new CMS at the time. But anyway yes @torinfo the customised name from the db would be good because then at least it can be changed once people see this in action. e.g. some might be inclined to change the name of the Xerte Online Toolkit template to just XOT especially where long sub template names are in use.

**Comment by FayCross** [comment]
Fixed in https://github.com/thexerteproject/xerteonlinetoolkits/commit/6da64a10bd9a23c262fa273b1bdc8e33ed4c0812
